src/js/angular/controllers.js

```diff
@@ -313,6 +314,8 @@ function mainCtrl($scope, $menuItems, $jwtAuth, $http, toastr, $location, $repos
         }
     });
 
+    $scope.urlResolver = DocumentationUrlResolver.getDocumentationUrl;
```

Comment:
I don't like exposing functions this way. It violates the encapsulation and is generally not testable. Better just transform this to a delegate function like
```
$scope.resolveUrl = () => DocumentationUrlResolver.getDocumentationUrl();
```

you can leave the urlResolver now, but I don't like it as a name because it's actually a function you call and not a class or something

Reply:
done